client/src/featureform/dashboard_metadata.py

```diff
@@ -381,9 +381,13 @@ def labels(rowData):
 
 
 def entities(rowData):
-    label_list = sqlObject.query_resource_variant(
-        "label_variant", "entity", rowData["name"]
-    )
+    try:
+        label_list = sqlObject.query_resource_variant(
+            "label_variant", "entity", rowData["name"]
+        )
+    except Exception as e:
+        print(f"No labels found for entity {rowData['name']} - {e}")
```

Comment:
should we throw an error? If yes, can we use the `LabelNotFound`?

Reply:
I'd say no because in the "valid" scenario where we don't have a label registered, the entity list will throw a 500 instead of gracefully returning an empty list.

Reply:
So it needs to be a "soft" exception as I understand the issue.